tests/suites/deploy/deploy_charms.sh

```diff
@@ -170,9 +170,7 @@ run_deploy_lxd_to_machine() {
 
 	ensure "${model_name}" "${file}"
 
-	juju add-machine -n 2 --series=jammy
-
-	charm=./tests/suites/deploy/charms/lxd-profile-alt
```

Comment:
why this line gets deleted? doesn't have anything to do with the series/base changes

Reply:
Oops